Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.9
```

Comment:
I also added a dockerfile in #11. There are gonna be merge conflicts. Shall I remove mine? Do you think we can also add "RUN dvc repro" in the image to get the artifacts before serving the model?

Reply:
I don't think a merge conflict for a dockerfile is that big of a deal. If you want to you can update that file with the version on this branch to avoid conflicts.

As for the `dvc repro` I am not sure what's the best approach here as I don't have a clear picture right now of how the pipeline looks like. I'm open to any suggestions.